### PR TITLE
Merge xeus python lite2

### PR DESCRIPTION
**Pull request by DerThorsten**

### CODE REVIEW

**Review by martinRenou — APPROVED** [review]
🚀 Thanks!

**Comment by jtpio** [comment]
Is there an issue to track the remaining items to be done after the incorporation in the main repo?

File: lite/.github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
+name: Build
```
Comment (jtpio) [inline lite/.github/workflows/build.yml:1]: These workflows should be hoisted to the top-level so they can actually run.

File: lite/docs/build-environment.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: xeus-python-kernel-docs
```
Comment (jtpio) [inline lite/docs/build-environment.yml:1]: Same as above, this should be merged with the existing docs.

File: lite/.github/workflows/build.yml (hunk 1 above)
Comment (martinRenou) [reply lite/.github/workflows/build.yml:1]: The work is being done in separate PRs against the [merge-xeus-python-kernel](https://github.com/jupyter-xeus/xeus-python/tree/merge-xeus-python-kernel) branch

**Comment by jtpio** [comment]
> Is there an issue to track the remaining items to be done after the incorporation in the main repo?

It's in https://github.com/jupyter-xeus/xeus-python/issues/596

**Pull request merged**